tests/pages/services/ServiceActions-cy.js

```diff
@@ -120,29 +120,24 @@ describe("Service Actions", function() {
 
       it("opens the correct service destroy dialog", function() {
         cy
-          .get(".confirm-modal p span")
+          .get(".modal-body p strong")
           .contains("/sleep")
           .should("to.have.length", 1);
       });
 
-      it("disables button during API request", function() {
-        cy.route({
-          method: "DELETE",
-          url: /marathon\/v2\/apps\/\/sleep/,
-          response: []
-        });
+      it("disable danger button while service name isn't correct", function() {
         cy
           .get(".confirm-modal .button-collection .button-danger")
-          .click()
           .should("have.class", "disabled");
       });
 
-      it("closes dialog on successful API request", function() {
+      it("closes dialog when user type correct service name", function() {
```

Comment:
@MatApple this is what is happening, I just change the wording because now we have to type the service name.